emmy.go

```diff
@@ -133,52 +135,71 @@ func runClients(n int, concurrently bool, protocolType, protocolVariant, endpoin
 // Parameters passed to the client in client.ProtocolParams struct have fixed
 // values for demonstration purposes.
 func runClient(protocolType, protocolVariant, endpoint string) {
-	genClient := getProtocolClient(protocolType, protocolVariant, endpoint)
-	if genClient == nil {
-		cLogger.Critical("Error ocurred when creating client")
+	_, pbVariant, err := parseSchema(protocolType, protocolVariant)
+	if err != nil {
+		cLogger.Criticalf("%v", err)
 		return
 	}
 
-	// Placeholder for values used to bootstrap the chosen protocol
-	// The actual (example-specific) values will be filled in
-	// depending on which protocol we are starting
-	protocolParams := client.ProtocolParams{}
+	var protocolErr error
 
 	switch protocolType {
 	case "pedersen":
-		protocolParams["commitVal"] = big.NewInt(121212121)
+		commitVal := big.NewInt(121212121)
```

Comment:
Is this dummy initialization still needed? If yes, we should probably move it into `NewPedersenClient` function. IMO such an initialization should never be needed so please double check if it can be avoided. Again, I might be missing something.